CL/cl_ext.h

```diff
@@ -2441,6 +2441,11 @@ typedef struct _cl_queue_family_properties_intel {
 #define CL_QUEUE_CAPABILITY_BARRIER_INTEL                   (1 << 25)
 #define CL_QUEUE_CAPABILITY_KERNEL_INTEL                    (1 << 26)
 
+/* cl_intel_queue_no_sync_operations */
```

Comment:
A few cosmetic suggestions:

* Can you expand this "header comment" to follow a similar format as the other extensions?  Right now it's easy to misinterpret these defines as a part of the preceding extension and not as a new extension.
* Can you add a comment describing which bitfield type `CL_QUEUE_NO_SYNC_OPERATIONS` is for?  I think it's an addition to `cl_command_queue_properties`?

We really should merge the header generation script (#161) so most of this will happen automatically...

Reply:
There's a weak process note about reserving enums in cl.xml, but nothing about bits AFAIK. Making that process stronger (Vulkan has a very detailed set of steps to be taken before consuming shared resources like bits) doesn't guarantee no collisions or assigning values from the wrong end, but it definitely helps a lot.